stock_available_immediately/models/product_product.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2014 Camptocamp, Akretion, Numérigraphe
+# Copyright 2016 Sodexis
+# Copyright 2019 Sergio Díaz <sergiodm.1989@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _compute_available_quantities_dict(self):
+        res, stock_dict = super()._compute_available_quantities_dict()
+        for product in self:
+            if not product.is_storable:
+                res[product.id]["immediately_usable_qty"] = stock_dict[product.id][
+                    "virtual_available"
+                ]
+            else:
```

Comment:
```suggestion
            if product.is_storable:
```

That's already covered by `super()`, isn't it?
See: https://github.com/OCA/stock-logistics-availability/blob/ea37c52d99f652abef09953b03bd17a320bf8150/stock_available/models/product_product.py#L47

Reply:
indeed. changes are done.